scripts/update_existing_dataset.R

```diff
@@ -13,9 +13,10 @@
 # - reupload metadata to postgres if necessary
 # =======================================================================
 
-# setup dataset class: (change `id` to the id of your pipeline)
-id <- "A1"
-ds <- statbotData::create_dataset(id)
+# setup dataset class: (change `data_indicator` to the data_indicator
+# of your dataset
+data_indicator <- "A1"
+ds <- statbotData::create_dataset(data_indicator)
```

Comment:
```suggestion
```
**suggestion:** Each pipeline script (e.g. `A1.R`) explicitely calls e.g. `create_dataset("A1")`. I think calling `create_dataset("A1")` here and then sourcing the pipeline is redundant. (the variable `ds` will be available after calling `source(path)`)